deebot_client/commands/json/custom_cut_mode.py

```diff
@@ -0,0 +1,21 @@
+"""Advanced mode command module."""
+
+from __future__ import annotations
+
+from deebot_client.events import CustomCutModeEvent
+
+from .common import GetEnableCommand, SetEnableCommand
+
+
+class GetCustomCutMode(GetEnableCommand):
```

Comment:
What does this option do?
Can you post a screenshot from this setting in the app

Reply:
Please answer this comment

Reply:
That's what I tried to find out in my test-setup. I've tried almost everything and it's not changeable via app setting, but it probably determines which options are available in the app. It is activated for the A1600 RTK in any case.

It may enable the "Enhanced" mode..

![image](https://github.com/user-attachments/assets/2e7fbd5a-fd0c-4ec6-9e16-72df2f8fe249)